tests/scripts/thread-cert/Cert_5_3_02_RealmLocal.py

```diff
@@ -96,14 +96,14 @@ def test(self):
                 self.assertTrue(self.nodes[LEADER].ping(addr, size=256))
                 self.assertTrue(self.nodes[LEADER].ping(addr))
 
-        self.assertTrue(self.nodes[LEADER].ping('ff03::1', size=256))
-        self.assertTrue(self.nodes[LEADER].ping('ff03::1'))
+        self.assertTrue(self.nodes[LEADER].ping('ff03::1', num_responses=2, size=256))
+        self.assertTrue(self.nodes[LEADER].ping('ff03::1', num_responses=2))
```

Comment:
That would mean enabling OpenThread to receive IPv6 multicast messages on the same device that sourced them.  I updated this PR with that feature enabled.